examples/units/basic_units.py

```diff
@@ -345,7 +348,20 @@ def convert(val, unit, axis):
         if units.ConversionInterface.is_numlike(val):
             return val
         if np.iterable(val):
-            return [thisval.convert_to(unit).get_value() for thisval in val]
+            if np.ma.isMaskedArray(val):
+                val = val.astype(float).filled(np.nan)
+            out = np.empty((len(val),),  dtype=float)
```

Comment:
the 1-tuple is not needed, `np.empty(len(val))` is good enough (or `np.empty(len(val), float)` is you really want to be explicit)

Reply:
Done